src/components/SeldonImage/SeldonImage.tsx

```diff
@@ -83,6 +115,8 @@ const SeldonImage = memo(
         alt,
         srcSet,
         sizes,
+        loading,
```

Comment:
should we have defaults for these?

Reply:
img itself default to `eager` and `auto` respectively